backend/bot/index.js

```diff
@@ -14,7 +29,23 @@ bot.on('start', async () => {
   }
 });
 
-const postResultToSlack = (match) => {
+const updateSlack = async (match) => {
+  // Update channel purpose
+  const users = await storage.getAllUsers(true);
+  const purposeCharacterLimit = 250;
+  let newChannelPurpose = '';
+  let i = 1;
+  for(const user of users) {
+    const purposeLine = `${i}. ${user.name} ${user.rating}\n`;
+    if(purposeLine.length + newChannelPurpose.length > purposeCharacterLimit) {
```

Comment:
why not empty space between "in" and "(" ???

Reply:
*`if`